src/lib/api/services/Auth-form.ts

```diff
@@ -8,29 +8,32 @@ import {
 
 export const authForm = apiSlice.injectEndpoints({
   endpoints: (builder) => ({
-    getAuthRegister: builder.mutation<void, RegisterAuthForm>({
+    register: builder.mutation<void, RegisterAuthForm>({
       query: (formData) => ({
         url: `/auth/register`,
         method: "POST",
         body: formData,
       }),
     }),
 
-    getAuthLogin: builder.mutation<void, LoginAuthForm>({
+    login: builder.mutation<void, LoginAuthForm>({
```

Comment:
`login: builder.mutation<LoginAuthForm, void>({`

and other too. sorry for reminding, it is just recommendation to do best practice bong.